examples/fscloud/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = ">= 1.60.0"
+      version = "1.62.0"
```

Comment:
This example is purposely using `>=` to test latest provider. There are other examples locked into lowest supported version

Reply:
ahh it's wobbly, we probably want to review this as depends on tribal knowledge in the team.

Reply:
its stated in the code comment above the logic

Reply:
Yes, but the comment does not really resonate as it is relatively long, talk about 1 example, and is relatively far from the line that is changes.

Reply:
I think this should be changed through the test stack- eg: have some dynamic change there on the version pinning so that there is fewer cognitive burden to understand / upgrade the example.

The core issue I think is that the examples as they are now are designed to be run in the test suite, which facilitate the job of the core team, but this is no really a matter for most consumers who are using those examples are a starting point / good example.